src/components/Observations/index.tsx

```diff
@@ -41,10 +67,12 @@ export default function PlantsDashboard(): JSX.Element {
       setPlantsSitePreferences={onPreferences}
       allowAllAsSiteSelection={true}
       onPlantingSites={onPlantingSites}
-      isEmptyState={!plantingSites?.length || !hasObservations}
+      isEmptyState={!plantingSites?.length || !observationsResults?.length}
     >
-      {hasObservations ? (
-        <div>placeholder for selected planting site {selectedPlantingSite?.id}</div>
+      {observationsResults === undefined && observationsResultsError === undefined ? (
+        <CircularProgress sx={{ margin: 'auto' }} />
+      ) : observationsResults?.length && selectedPlantingSite ? (
```

Comment:
i have to update this,, show empty state if a specific selected planting site has no observations.